Apps/WebClient/src/ClientApp/src/views/registration.vue

```diff
@@ -681,13 +681,17 @@ export default class RegistrationView extends Vue {
                 </b-form>
                 <div v-else-if="!clientRegistryError">
                     <h1>Minimum age required for registration</h1>
-                    You must be <strong>{{ minimumAge }}</strong> years of age
-                    or older to use this application
+                    <p data-testid="minimumAgeErrorText">
+                        You must be <strong>{{ minimumAge }}</strong> years of
+                        age age or older to use this application
```

Comment:
there is a repeat of "age", there should only be 1.

Reply:
fixed